README.md

```
@@ -141,7 +148,28 @@ Assuming your branch is `feature/scw`, and your feature is targeted for release
-* Read me updates: These are not required to be on a branch and can be committed directly to main.
-* Bug fixes that can be OTA updated: These can be committed directly to main to perform an OTA update.
+
+- Build scripts: These may need to be run off of the default main branch instead of feature or release branches.
+- Read me updates: These are not required to be on a branch and can be committed directly to main.
+- Bug fixes that can be OTA updated: These can be committed directly to main to perform an OTA update.
+
+## Firebase Configuration
+
+Firebase config files are not stored in this `git` repo for security. New team members need to:
+
+1. Obtain the Firebase config files from the private repository at https://github.com/ephemeraHQ/converse-app-env
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)